src/main/java/io/debezium/connector/spanner/db/DatabaseClientFactory.java

```diff
@@ -96,6 +97,15 @@ else if (credentialsPath != null) {
                 LOGGER.error(e.getMessage(), e);
             }
         }
+        else {
```

Comment:
This method is for testing. Do you have any test for that? If not then the code should not be added.

Reply:
https://github.com/debezium/debezium-connector-spanner/pull/65/commits/32a1f61b6c65971377dd22f4a7d124915b2bb1a2
I added a test case when credentialJson and credentialFile are null. I am not sure if it is a good idea to add such a test case. Please let me know if there is an appropriate test case.